presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -120,19 +121,19 @@ public void testShowStatsWithoutFromFails()
     @Test
     public void testShowStatsWithMultipleFromFails()
     {
-        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, lineitem)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
+        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, nation_partitioned)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
     }
 
     @Test
     public void testShowStatsWithGroupByFails()
     {
-        assertQueryFails("SHOW STATS FOR (SELECT avg(totalprice) FROM orders GROUP BY clerk)", ".*GROUP BY is not supported in SHOW STATS SELECT clause");
+        assertQueryFails("SHOW STATS FOR (SELECT avg(totalprice) FROM orders GROUP BY orderkey)", ".*GROUP BY is not supported in SHOW STATS SELECT clause");
     }
 
     @Test
     public void testShowStatsWithHavingFails()
     {
-        assertQueryFails("SHOW STATS FOR (SELECT avg(orderkey) FROM orders HAVING avg(orderkey) < 5)", ".*HAVING is not supported in SHOW STATS SELECT clause");
+        assertQueryFails("SHOW STATS FOR (SELECT categoryid FROM orders GROUP BY categoryid HAVING categoryid < 5)", ".*HAVING is not supported in SHOW STATS SELECT clause");
```

Comment:
unrelated change

Reply:
It it related to code refactoring, see answer to @arhimondr 's question "Why did you pull it up here?"